cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -255,8 +257,7 @@ macro (build_addon target prefix libs)
 
         if(CMAKE_BUILD_TYPE MATCHES Debug)
           # for debug builds also install the PDB file
-          get_filename_component(LIBRARY_DIR ${LIBRARY_LOCATION} DIRECTORY)
-          install(FILES ${LIBRARY_DIR}/${target}.pdb DESTINATION ${target}
+          install(FILES ${PROJECT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/${target}.pdb DESTINATION ${target}
                   COMPONENT ${target}-${${prefix}_VERSION})
         endif()
```

Comment:
What exactly is broken/failing? It's hard for me to test Windows/VS. Not my platform. I can try to test during the weekend though.